src/ansys/additive/core/microstructure.py

```diff
@@ -213,123 +230,130 @@ def material(self, value):
 
     @property
     def sample_min_x(self) -> float:
-        """Minimum x coordinate (m) of the geometry sample."""
+        """Minimum x coordinate of the geometry sample (m).
+
+        Valid values are from :obj:`MIN_POSITION_COORDINATE` to :obj:`MAX_POSITION_COORDINATE`.
+        """
         return self._sample_min_x
 
     @sample_min_x.setter
     def sample_min_x(self, value: float):
         self.__validate_range(
-            value, self.__MIN_POSITION_COORDINATE, self.__MAX_POSITION_COORDINATE, "sample_min_x"
+            value, self.MIN_POSITION_COORDINATE, self.MAX_POSITION_COORDINATE, "sample_min_x"
         )
         self._sample_min_x = value
 
     @property
     def sample_min_y(self) -> float:
-        """Minimum y coordinate (m) of the geometry sample."""
+        """Minimum y coordinate of the geometry sample (m).
+
+        Valid values are from :obj:`MIN_POSITION_COORDINATE` to :obj:`MAX_POSITION_COORDINATE`.
+        """
         return self._sample_min_y
 
     @sample_min_y.setter
     def sample_min_y(self, value: float):
         self.__validate_range(
-            value, self.__MIN_POSITION_COORDINATE, self.__MAX_POSITION_COORDINATE, "sample_min_y"
+            value, self.MIN_POSITION_COORDINATE, self.MAX_POSITION_COORDINATE, "sample_min_y"
         )
         self._sample_min_y = value
 
     @property
     def sample_min_z(self) -> float:
-        """Minimum z coordinate (m) of the geometry sample."""
+        """Minimum z coordinate of the geometry sample (m).
+
+        Valid values are from :obj:`MIN_POSITION_COORDINATE` to :obj:`MAX_POSITION_COORDINATE`.
+        """
         return self._sample_min_z
 
     @sample_min_z.setter
     def sample_min_z(self, value: float):
         self.__validate_range(
-            value, self.__MIN_POSITION_COORDINATE, self.__MAX_POSITION_COORDINATE, "sample_min_z"
+            value, self.MIN_POSITION_COORDINATE, self.MAX_POSITION_COORDINATE, "sample_min_z"
         )
         self._sample_min_z = value
 
     @property
     def sample_size_x(self) -> float:
         """Size of the geometry sample in the x direction (m).
 
-        Valid values are from 0.001 to 0.01.
+        Valid values are from :obj:`MIN_SAMPLE_SIZE` to :obj:`MAX_SAMPLE_SIZE`.
+        When setting `sample_size_x`, the value must be greater than `sensor_dimension`
+        plus :obj:`MIN_XY_SIZE_CUSHION`.
```

Comment:
```suggestion
        When setting the```sample_size_x`` parameter, the value must be greater than the ``sensor_dimension``
        value plus the :obj:`MIN_XY_SIZE_CUSHION` value.
```